apps/files/src/store/mutations.js

```diff
@@ -141,7 +141,29 @@ export default {
     state.trashbinDeleteMessage = message
   },
   SET_FILES_DELETE_CONFIRMATION (state, message) {
-    state.filesDeleteMessage = message
+    state.deleteDialogMessage = message
+  },
+  PROMPT_FILE_RENAME (state, file) {
+    state.renameDialogOpen = true
```

Comment:
rename dialog is now kind of encapsulated here in the global store instead of a mixin and wrapped inside action/mutation

in the future we should move this to a separate store module, maybe one dedicated to dialogs or file actions